Fix Search customer in specific group

khouloudbelguith opened the pull request:
<!--
Thank you for contributing to the PrestaShop project! 

Please take the time to edit the "Answers" rows with the necessary information.

Check out our contribution guidelines to find out how to complete it:
https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/#pull-requests
 -->

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Navigate to a customer group which has more than one customer, then try to search with a name. On click "Search" you have to redirect to list of groups ...not having search results. Going back to the list of customers in the same group, search filter works OK, reset button same work.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9825 
| How to test?  | Navigate to a customer group which has more than one customer, then try to search with a name. On click "Search" => Ok ( Same test with reset)

<!-- Click the form's "Preview" button to make sure the table is functional in GitHub. Thank you! -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/11085)
<!-- Reviewable:end -->

khouloudbelguith committed:
Fix Search customer in specific group

mickaelandrieu commented:
Hello @khouloudbelguith,

you can fix the coding style by executing this command and commit the updated files:

```
./vendor/bin/php-cs-fixer fix
```

Mickaël

Commit:
fix the coding style

khouloudbelguith commented:
Hi @mickaelandrieu,

Thanks for your review!
Done.

Commit:
Fix issue with the link Manage tax rules in the pricing tab

SebBareyre committed:
Add breadcrumb to the new products page

add the missing breadcrumb function

SebBareyre committed:
Add breadcrumb to the prices drop page

add the missing breadcrumb function

SebBareyre committed:
Add breadcrumb to the best sales page

add the missing breadcrumb function

SebBareyre committed:
Add breadcrumb to the suppliers page

add the missing breadcrumb function

unlocomqx committed:
Match multiline translation expressions in modules

zuk3975 committed:
Initial commit. Controller and empty index view

zuk3975 committed:
Renders list of empty categories

zuk3975 committed:
Implements empty categories list actions and filtering

zuk3975 committed:
Prefixed ns with /Monitoring

zuk3975 committed:
Renders products with combinations list without actions

zuk3975 committed:
adds query missing query parts in QueryBuilder

zuk3975 committed:
Adds AbstractProductQueryBuilder and renders ProductWithoutCombination list

zuk3975 committed:
Renders disabled products list

zuk3975 committed:
Renders productWithoutImage grid

zuk3975 committed:
Renders productWithoutDescription grid

zuk3975 committed:
Renders products without price grid

zuk3975 committed:
Adds filter to reference field in products lists

zuk3975 committed:
Enabled javascript for product lists

zuk3975 committed:
Adds filters request parsing in controller

zuk3975 committed:
Minor code optimization

zuk3975 committed:
Renames method in controller

zuk3975 committed:
Optimizes code

zuk3975 committed:
Adds security annotation to index action

zuk3975 committed:
Fixes camel case convention controller to twig vars

YeLnatSs committed:
Fix on submenu which hides menu

zuk3975 committed:
Minor code optimization

YeLnatSs committed:
Fix on submenu hiding menu

YeLnatSs committed:
Build assets

zuk3975 committed:
Explicates grid names

zuk3975 committed:
Implements abstract product grid definition

Commit:
Update CLDR data to 35.1

idnovate committed:
Execute displayAdminLogin hook without shop context

Commit:
Update CLDR test fixtures

Commit:
Fix test using hex values (to avoid charset problems)

zuk3975 committed:
Builds monitoring assets

sarjon committed:
Start working on behat tests for Category

sarjon committed:
Add test for adding new category

zuk3975 committed:
Adds Distinct to count queries

jolelievre reviewed (dismissed):
Thank you @khouloudbelguith

PierreRambaud commented on controllers/admin/AdminGroupsController.php:
```diff
@@ -186,6 +186,10 @@ public function initProcess()
                 $this->processResetFilters();
             }
+
+            if (Tools::getIsset('submitFilter')) {
```
Why not also check for `Tools::getIsset('id_group')`?

PierreRambaud replied:
And should it be?
```suggestion
            if (Tools::isSubmit('submitFilter')) {
```

Quetzacoalt91 committed:
Merge pull request #12997 from francoispeyret/develop

[+] BO: add disable visual effect on translations

Quetzacoalt91 committed:
Merge pull request #14562 from jolelievre/update-changelog

Update Changelog for v1.7.6.0 after RC2

Commit:
Update command for lint in package.json

Commit:
Update readme

Commit:
modify name of report file to add hours:minutes:seconds

Commit:
Delete comments on Dockerfile

Commit:
Add package-lock.json

Commit:
Merge branch 'addingE2E-puppeteer' of https://github.com/mbadrani/PrestaShop into addingE2E-puppeteer

boubkerbribri committed:
Update README

Co-Authored-By: GoT <PierreRambaud@users.noreply.github.com>

boubkerbribri committed:
Update README

Co-Authored-By: GoT <PierreRambaud@users.noreply.github.com>

boubkerbribri committed:
Update README

Co-Authored-By: GoT <PierreRambaud@users.noreply.github.com>

Commit:
Remove image in about us cms page

Commit:
Update comment in docker-run.sh

Commit:
Merge branch 'addingE2E-puppeteer' of https://github.com/mbadrani/PrestaShop into addingE2E-puppeteer

matks committed:
Merge pull request #13983 from pumamood/develop

Fix empty tag in webservice XML response

matthieu-rolland committed:
Merge pull request #14545 from jolelievre/remove-mails

Remove english emails since they are generated now

zuk3975 committed:
Modifies list qb to fix showing products without combinations

zuk3975 committed:
Removed status from disabled products list

zuk3975 committed:
Adds missing docblock

PierreRambaud committed:
Merge pull request #14560 from PierreRambaud/fix/typo

Typo fix on bug issue template

PierreRambaud committed:
Merge pull request #14529 from PierreRambaud/fix/14504

Fix position problem on a fresh install

PierreRambaud committed:
Merge pull request #14567 from marionf/issue-14559

Remove image in about us cms page

Progi1984 committed:
Implement Schema ItemList for products listing & Fix Schema Product (Fix Selenium tests)

Commit:
adding first version of naming conventions document

matks committed:
Merge branch '1.7.6.x' of github.com:PrestaShop/PrestaShop into merge176xtodevelop-33

Commit:
fixing CONVENTIONS

Commit:
fix typo in markdown (titles)

Commit:
Update text in ReadMe file

Commit:
Replace console.log with err or warn in some cases

Commit:
Many things that were supposed to be removed weren't:

- Anything under admin, because the folder was renamed before cleaning it up
- Legacy tests
- .estlintignore, .eslintrc.js, .php_cs.dist, .docker-compose.yml, .webpack, .composer-lock, .package-lock.json...

Commit:
Disable two removals temporarily

Commit:
Fix many things that were supposed to be removed during build not being removed (#14579)

Fix many things that were supposed to be removed during build not being removed

PierreRambaud committed:
Merge pull request #14576 from matks/merge176xtodevelop-33

Merge 1.7.6.x to develop - 10/07/2019

mbadrani committed:
Merge pull request #14508 from mbadrani/addingE2E-puppeteer

Integrate Puppeteer to check links on FO and BO

Commit:
add proper example

Commit:
add option to force log passed request (disabled by default)

matthieu-rolland committed:
upgrade search_word table

matthieu-rolland committed:
fix search with long words:

- increase search_word's word column max size
- increase search word's max size in the code

Commit:
Merge pull request #14525 from 202-ecommerce/patch-7

Fix php warning in php 7.2 in case of JSON output

Commit:
Merge pull request #14530 from PierreRambaud/fix/14054

Use Preview link without  option for seo preview

matks committed:
Merge pull request #14249 from tomas862/bugfix/hook-dispatcher

Case-insensitive hook dispatching

matks committed:
Merge branch '1.7.6.x' of github.com:PrestaShop/PrestaShop into merge176xtodevelop-34

Commit:
added mocha, chai and chai-string

Commit:
added configuration for mocha

Commit:
first POM implementation

Commit:
first test using mocha and POM

Commit:
fix syntax and methods not properly organized

jolelievre committed:
Merge pull request #14592 from matks/merge176xtodevelop-34

Merge 1.7.6.x to develop - 11/07/2019

jolelievre committed:
Revert removed templates for mail dynamic contents, order mails mainly, create MailPartialTemplateRenderer to manage this feature

Commit:
move the getPageTitle method to be available everywhere

Commit:
add comments in main scenario

matks committed:
Merge pull request #14320 from tomas862/features/min-max-form-type

Introduces NumberMinMaxFilterType and IntegerMinMaxFilterType

jolelievre committed:
Update MailVariablesTransformation to avoid using a hook in preview

Commit:
fix typo

Commit:
remove useless comment

Commit:
add require of globals.js in linkechecker to centralize parameters recuperation

Commit:
modified link in package.json

Commit:
Add argument to run shop in production mode

Commit:
Update dockerfile to match the latest one on puppeteer github

Commit:
Update command and volumes in docker-compose file

Commit:
Add command to run linkchecker script

Commit:
Add run-tests file to run command on package.json Update how tu run with docker-compose on readme

Commit:
Create reports folder if it not exist in linkchecker

matthieu-rolland committed:
fix broken translation in module list page:

- link module actions to related translation domain
- set those relations as module attribute
- fix module action display name generation
- use correct translation domain

Commit:
Fix space (eslint)

Commit:
Update comments in linkchecker

matks committed:
Merge pull request #13529 from zuk3975/m/list/monitoring

Migrate Sell > Catalog > Monitoring listing action

Progi1984 committed:
Implement Schema ItemList for products listing & Fix Schema Product (Factorize in a partial)

matthieu-rolland committed:
add translation for bootstrap file upload fields

PierreRambaud committed:
Merge pull request #14246 from sarjon/filters-applicator

Add doctrine filters applicator

PierreRambaud reviewed (changes requested):
See my comment

Commit:
Cs fixer

Commit:
Cs fixer

Commit:
Cs fixer

Commit:
Cs fixer

MathiasReker committed:
Update CurrencyTest.php

MathiasReker committed:
Update LocaleTest.php

MathiasReker committed:
Update LocaleTest.php

MathiasReker committed:
Update LocaleTest.php

MathiasReker committed:
Update LocaleTest.php

MathiasReker committed:
Update CurrencyTest.php

MathiasReker committed:
Update CurrencyTest.php

MathiasReker committed:
Update RepositoryTest.php

MathiasReker committed:
Update LocaleTest.php

MathiasReker committed:
Update LocaleTest.php

MathiasReker committed:
Update LocaleTest.php

Commit:
Fix France formatted number in tests

marekhanus committed:
WS: Auto generate link_rewrite and do not require it when creating categories

MathiasReker committed:
Update AbstractTypedCollectionTest.php

MathiasReker committed:
Update MailTemplateTwigRendererTest.php

MathiasReker committed:
Update AbstractTypedCollectionTest.php

marekhanus committed:
PrettyCI missing ,

PierreRambaud committed:
Merge pull request #14231 from MathiasReker/test

php unit no expectation annotation

sarjon committed:
Explain hardcoded data

sarjon committed:
Apply CS fixer

Commit:
Install google-chrome-stable version and unstable one

zuk3975 committed:
Moves setShopAssoc method after update action in manufacturer and category edit handlers

sarjon committed:
Added Behat tests for customer required fields

PierreRambaud committed:
Merge pull request #14612 from boubkerbribri/updateDockerInPuppeteer

Update docker for puppeteer

sarjon committed:
Add Behat tests for setting private note about customer

Commit:
use DI when instantiating pages and change the way we handle browser and pages instantiation

Commit:
change the name of the superclass Page to be less confusing (already used in puppeteer)

Commit:
change convention to reflect what we decided relative to selectors names

Commit:
change beforeFunc function name and package.json

jolelievre committed:
Better injection for MailPartialTemplateRenderer, LangueCore implements LanguageInterface

jolelievre committed:
Fix service declaration

jolelievre committed:
Update version to 1.7.7.0

matthieu-rolland committed:
Merge pull request #14587 from matthieu-rolland/fix-search-long-words

Fix issue when searching for long words in FO

matks committed:
Merge pull request #14691 from jolelievre/update-version-177

Update version to 1.7.7.0

matks committed:
Merge pull request #14089 from 202-ecommerce/patch-3

Fix #14085 Shop url with port in header HOST

matks committed:
Merge pull request #14466 from rokaszygmantas/restore_save_employee_lang

Fix saving of selected BO language

matks committed:
Merge pull request #14613 from matthieu-rolland/fix-broken-translation

Broken translation in module list page:

matks committed:
Merge pull request #14540 from webmak/patch-1

Update AdminImportController.php

PierreRambaud committed:
Merge pull request #8769 from marekhanus/feature/category-link-rewrite

WS: Auto generate link_rewrite and do not require it when creating ca…

Commit:
add mochawesome folder to gitignore

Commit:
add mochawesome reporting plugin

Commit:
add mochawesome reporter in mocha launch options

matthieu-rolland committed:
improvement: make last breadcrumb element not clickable

zuk3975 committed:
Adds missing typecast in handler

Commit:
add mocha to the linter

Commit:
modify what linter asked

Commit:
add window size and linter recommandations

matks committed:
Remove codacy badge as we dont use it anymore

matthieu-rolland committed:
Merge pull request #14543 from matthieu-rolland/fix-stock-img

Wrong path of products thumbnails in back office

sarjon committed:
Fix display of tax incl/excl

Commit:
Add a parse to the HEADLESS option to ensure proper boolean interpretation

sarjon committed:
Add missing Discounts block

sarjon committed:
Catch all exceptions in controller

matks committed:
Move travis scope from 5.6 to 7.1

PierreRambaud committed:
Merge pull request #14287 from sarjon/category-behat-tests

Behat tests for Category

PierreRambaud committed:
Merge pull request #14214 from matks/update-error-cart-out-of-stock-msg-2

Handle shoppingcart error returns properly - part 2

PierreRambaud committed:
Merge pull request #14352 from jocel1/fix-customer-switch

move secure_key set at the right place

mbadrani committed:
Merge pull request #14593 from SimonGrn/POM

First POM implementation

PierreRambaud committed:
Merge pull request #14558 from Progi1984/issue12687

Implement Schema ItemList for products listing & Fix Schema Product

PierreRambaud committed:
Merge pull request #14307 from zuk3975/fix/brand-image

Fix manufacturers grid logo to not depend from shop context

PierreRambaud committed:
Merge pull request #14725 from matks/remove-travis-56

Move travis scope from 5.6 to 7.1

Commit:
Fix screenshot in readme, add new one

Fixes #14749

PierreRambaud committed:
Merge pull request #13677 from YeLnatSs/submenu-bug

Submenu hides menu

Commit:
Install missing language before running tests

PierreRambaud committed:
Merge pull request #12880 from SebBareyre/patch-44

Add breadcrumb to the new products page

PierreRambaud committed:
Merge pull request #12881 from SebBareyre/patch-45

Add breadcrumb to the prices drop page

PierreRambaud committed:
Merge pull request #12882 from SebBareyre/patch-46

Add breadcrumb to the best sales page

PierreRambaud committed:
Merge pull request #12883 from SebBareyre/patch-47

 Add breadcrumb to the suppliers page

Commit:
Add global args and fix some lint problems

Commit:
Add slowMo and Define language to start navigator

Commit:
Adding common functions needed

Commit:
Add object page Install used for tests install

Commit:
Adding Install prestashop test

Commit:
Deleting taking screenshot

PierreRambaud committed:
Merge pull request #14656 from zuk3975/m/brand-fix-shop-assoc

Fix editing shop association in Category and Manufacturer

Commit:
Update default values in global

Commit:
Change the way to check System compatibility step

matks committed:
Handle Add to Cart failure usecase where previous cart rules were empty or previous cart not available

Commit:
Update Page titles

Commit:
Adding some selectors checks in FO

matthieu-rolland committed:
Add category to product CQRS command

- add category to product CQRS command
- use the command in categoryController

Commit:
Add some commands to execute tests

Commit:
Fix URL FO in docker-compose

Commit:
Update default db password in globals

Commit:
Update function elementVisible

Commit:
Delete click on input

Commit:
Adding TestPath to run a specific test in docker

Commit:
Update ReadMe to include smoke tests

Commit:
Update readme

Commit:
Update URL_BO in docker-compose

Commit:
Update name of selector

Commit:
Update name of selectors

matks committed:
Merge pull request #13712 from sarjon/m/orders/view

Prepares Command API for migrating Order view page

matthieu-rolland committed:
Merge pull request #14764 from boubkerbribri/smokeInstall

Install prestashop test with puppeteer

Commit:
first version of the BO crawling smoke test

Commit:
fix problems with navigating too fast

Commit:
add lint corrections

Thomas-lhuillier committed:
Fix create account sitemap link

matthieu-rolland committed:
Merge pull request #14815 from SimonGrn/test_BO

Add a new smoke test: crawling BO

sarjon committed:
Attempt to fix inconsistent behavior

Commit:
fix merge suggestions
change default value for HEADLESS mode

matthieu-rolland committed:
Merge pull request #14825 from SimonGrn/fix_pom_bo

Fix some minor problems on the "crawling BO" smoke test

matks committed:
Merge pull request #14718 from matks/remove-codacy-badge

Remove codacy badge as we dont use it anymore

PierreRambaud committed:
Merge pull request #13980 from idnovate/patch-4

Execute displayAdminLogin hook without shop context

PierreRambaud committed:
Merge pull request #12497 from khouloudbelguith/fix-issues-12475

Fix issue with the link Manage tax rules in the pricing tab

PierreRambaud committed:
Use a better way to install languages

PierreRambaud committed:
Merge pull request #14819 from Thomas-lhuillier/patch-1

Fix create account sitemap link

PierreRambaud committed:
Merge pull request #12903 from unlocomqx/translate

Match multiline translation expressions in modules

matks committed:
Merge pull request #14065 from sarjon/m/carts/view

Migrates Shopping Carts view page

matks committed:
Merge pull request #14670 from sarjon/customer-behat-tests

Add Behat tests for Customer commands

PierreRambaud committed:
Move i18n.prestashop.net into https and remove unwanted file

matks committed:
Re-apply PR 13849

PierreRambaud committed:
Merge pull request #13906 from eternoendless/update-cldr-35.1

Update CLDR data to 35.1

Progi1984 committed:
Initialize container in WebService environment

Progi1984 committed:
Fix review commments

matthieu-rolland committed:
add type hinting and use symfony response codes constant

matthieu-rolland committed:
use response object's http bad request constant

Progi1984 committed:
Fix review commments (assert in unit tests)

Progi1984 committed:
Fix error on assignment

PierreRambaud committed:
Merge pull request #13830 from davidglezz/patch-2

Remove duplicate var in cart.php

PierreRambaud committed:
Merge pull request #14710 from matthieu-rolland/feat-breadcrumb-seo

Make last breadcrumb element not clickable

PierreRambaud committed:
Merge pull request #13730 from 123monsite-regis/patch-26

Admin Specific Price : wrong display

Progi1984 committed:
Fix some code review comments

Progi1984 committed:
Add an exception when SymfonyContainer is not initialized

Progi1984 committed:
Move $context to the constructor and not anymore in parameter

Progi1984 committed:
Fix Pretty CI

PierreRambaud committed:
Merge pull request #14872 from matks/remove-AdminCmsController

Delete migrated AdminCmsController again

PierreRambaud committed:
Merge pull request #14859 from PierreRambaud/https-and-remove-cldr

Move i18n.prestashop.net into https and remove unwanted file

PierreRambaud committed:
Merge pull request #14800 from matthieu-rolland/improve-command-bus

Add category to product CQRS command

PierreRambaud committed:
Merge pull request #14601 from jolelievre/fix-order-conf

Revert removed templates for mail dynamic contents, order mails mainl…

PierreRambaud committed:
Merge pull request #14899 from Progi1984/issue14842_176x_bis

Initialize container in WebService environment (bis)

zalexki committed:
Update Module.php

matthieu-rolland committed:
Merge pull request #14949 from zalexki/patch-1

Update Module.php docs on version property type

Commit:
Merge branch 'fix-Search-customer-in-specific-group' of https://github.com/khouloudbelguith/PrestaShop into fix-Search-customer-in-specific-group

Commit:
change getIsset to isSubmit

khouloudbelguith commented:
Hi @PierreRambaud, @marionf this PR is replaced by this https://github.com/PrestaShop/PrestaShop/pull/14977

Thanks!
